examples/mnist/mnist_patch.py

```diff
@@ -4,12 +4,14 @@
 """
 
 import os
-import subprocess as sp
 
 import torch
 from torchvision.datasets.mnist import MNIST, read_image_file, read_label_file
 from torchvision.datasets.utils import extract_archive
 
+# import subprocess as sp
```

Comment:
Do we still need to this file actually ?

Reply:
I think we still need a patch, since `download` performs archive processing :
https://github.com/pytorch/vision/blob/01dfa8ea81972bb74b52dc01e6a1b43b26b62020/torchvision/datasets/mnist.py#L149-L164

Reply:
I see, can't we put it into GA action as a script or so ?

Reply:
Sure! we can try, i think it will be nice addition, I'll test it locally